inf-clojure.el

```diff
@@ -685,6 +730,20 @@ If you are using REPL types, it will pickup the most approapriate
   :type 'string
   :package-version '(inf-clojure . "2.0.0"))
 
+(defcustom inf-clojure-set-ns-form-planck
+  "(in-ns '%s)\n"
```

Comment:
Right, Planck's `in-ns` is a REPL special (just like in the ClojureScript REPL). It has not namespace.